ouroboros-consensus/test-storage/Test/Ouroboros/Storage/LedgerDB/OnDisk.hs

```diff
@@ -469,10 +469,10 @@ runMock cmd initMock =
         -- The snapshot that the real implementation will write to disk
         --
         -- The real implementation keeps the length of the blocks between @k@
-        -- and @k + snapEvery 1@ (provided that there are enough blocks). The
-        -- function 'ledgerDbCountToPrune' computes how many (old) blocks should
-        -- be dropped to get back into that range; the last (most recent) block
-        -- to be dropped will become the new anchor. It is the anchor that gets
+        -- and @k + 1@ (provided that there are enough blocks). The function
```

Comment:
Is `k + 1` correct? Oh, is that after pushing a block before `ledgerDbCountToPrune`?

Reply:
Yes, it's actually k. I bit confusing, because the implementation uses this function for the in memory snapshots while here it's used to create a disk snapshot.